react/features/base/media/components/web/Video.js

```diff
@@ -134,7 +137,12 @@ type Props = {
     /**
      * The value of the muted attribute for the underlying video element.
      */
-    muted?: boolean
+    muted?: boolean,
+
+    /**
+     * The virtual background object.
+     */
+    virtualBackground?: Object
```

Comment:
This shouldn't really be here. You have access to the track so this should be doable without adding such high level logic to a low level component. Did you run into any problems?